src/App.js

```diff
@@ -1,48 +1,29 @@
 import React, { Component } from 'react';
-import {Route, Switch} from 'react-router-dom';
-import {bindActionCreators} from 'redux';
-import {connect} from 'react-redux';
+import { Route, Switch } from 'react-router-dom';
 
-import {fetchMovies} from './actions/movies';
+import Navbar from './components/navbar';
+import SearchView from './views/searchView';
 
 import logo from './logo.svg';
 import './App.css';
 
-class App extends Component {
-  componentWillMount() {
-    this.props.fetchMovies();
-  }
+
+export default class App extends Component {
 
   render() {
     return (
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
+          <h1 className="App-title">Witaj w wyszukiwarce filmów</h1>
```

Comment:
That's nice - just please stay consistent to single language. ;)
English is ok ;)